doc/cascadia/SettingsSchema.md

```diff
@@ -89,54 +89,56 @@ Properties listed below are specific to each custom key binding.
 
 Bindings listed below are per the implementation in `src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp`
 
-- copy
-- copyTextWithoutNewlines
-- paste
-- newTab
-- openNewTabDropdown
-- duplicateTab
-- newTabProfile0
-- newTabProfile1
-- newTabProfile2
-- newTabProfile3
-- newTabProfile4
-- newTabProfile5
-- newTabProfile6
-- newTabProfile7
-- newTabProfile8
-- closeWindow
-- closeTab
-- closePane
-- switchToTab
-- nextTab
-- prevTab
-- increaseFontSize
-- decreaseFontSize
-- resetFontSize
-- scrollUp
-- scrollDown
-- scrollUpPage
-- scrollDownPage
-- switchToTab0
-- switchToTab1
-- switchToTab2
-- switchToTab3
-- switchToTab4
-- switchToTab5
-- switchToTab6
-- switchToTab7
-- switchToTab8
-- openSettings
-- splitPane
-- resizePaneLeft
-- resizePaneRight
-- resizePaneUp
-- resizePaneDown
-- moveFocusLeft
-- moveFocusRight
-- moveFocusUp
-- moveFocusDown
-- toggleFullscreen
+| Action | Description |
+| -------- | ----------- |
+| `copy` | Make a duplicate of the selected content. |
+| `copyTextWithoutNewlines` | Make a duplicate of the selected content and discard the newline characters. |
```

Comment:
```
    "CopyAction": {
      "description": "Arguments corresponding to a Copy Text Action",
      "allOf": [
        { "$ref": "#/definitions/ShortcutAction" },
        {
          "properties": {
            "action": { "type": "string", "pattern": "copy" },
            "copyNewlines": {
              "type": "boolean",
              "default": true,
              "description": "If false, concatenates the copied text to one line."
            }
          }
        }
      ]
    },
```
This one is _very temporarily_ blocked by #3811 but that should be going in soon so I wouldn't worry about it.